packages/antd/package.json

```diff
@@ -76,38 +76,17 @@
     "rc-picker": "2.7.6"
   },
   "devDependencies": {
-    "@ant-design/icons": "^5.6.1",
-    "@babel/cli": "^7.23.9",
```

Comment:
fyi, if we switch to pnpm then we may still need to explicitly list the dependencies we are using in each package